src/v2/controllers/FormController.js

```diff
@@ -251,7 +251,11 @@ export default Controller.extend({
     } else if (error.errorSummary) {
       errorObj = { responseJSON: error };
     }
-    model.trigger('error', model, errorObj || { responseJSON: { errorSummary: String(error) } }, true);
+    const showErrorBanner = !form?.isErrorMessageCustomized(errorObj);
+    model.trigger('error', model, errorObj || { responseJSON: { errorSummary: String(error) } }, showErrorBanner);
+    if (!showErrorBanner) {
+      form.showCustomErrorMessage(errorObj);
```

Comment:
should rename to `showCustomErrorCallout`  or `showCustomErrorBanner` ?

Reply:
renamed to `showCustomErrorCallout`